src/specfuzz_rtl.S

```diff
@@ -190,6 +192,9 @@ simulation_id:                          .quad   0
 debug_rollback_depth:                   .string     "[SF] rlbk: 0x%llx %lld %lld %lld\n"
 #endif
 
+.globl in_rlbk
```

Comment:
As it's a global variable, rename it to make clear that it belongs to specfuzz. Something like `specfuzz_executing_rollback` or at least `specfuzz_in_rlbk`

Reply:
Good idea.